core/app/assets/stylesheets/refinery/sections/_menu_item_icons.scss

```diff
@@ -0,0 +1,9 @@
+.site-name {@include icon('bars', $plugin_icon_colour, $margin: 1.2em)}
+
+.refinery_images_icon {@include icon('photo', $plugin_icon_colour, $margin: 1.2em)}
```

Comment:
I remembered we already started this conversation, so I guess you've answered my question here: https://github.com/refinery/refinerycms/pull/3167#issuecomment-222154804

I just reviewed it.  Here's my summary:

1. The <i></i> tag is superfluous markup
1. The erb is cleaner
1. Easier to change icons
1. The markup and the code is cleaner
1. A single place to see which icons are being used and how to use them
1. Mixins offer consistency and flexibility when generating icons

@anitagraham are these your reasons?  @bricesanchez you think this PR solves all of these?